service/project.go

```
@@ -1009,3 +1015,87 @@ func verifyAliasExists(alias, projectId string, newAliasDefinitions []model.Proj
 	}
 	return false, nil
 }
+
+func bbProjectIsValid(projName string, proj evergreen.BuildBaronSettings) error {
+	webHook, _, _ := model.IsWebhookConfigured(projName, "")
```

I think we should return an error if there's an error here.